server/views/applications/pages/health-needs/communication-and-language.njk

```diff
@@ -3,8 +3,131 @@
 
 {% block questions %}
 
-  <p class="govuk-body">
-    Communication and language needs page
-  </p>
+  {% set communicationFollowUp %}
+  {{
+      formPageTextArea(
+        {
+          fieldName: 'communicationDetail',
+          label: {
+            text: page.questions.hasCommunicationNeeds.communicationDetail.question,
+            classes: "govuk-label--s"
+          }
+        },
+        fetchContext()
+      )
+    }}
+  {% endset %}
+
+  {% set interpretationFollowUp %}
+  {{
+      formPageTextArea(
```

Comment:
I think this should be a `formPageInput` according to the design

Reply:
well spotted, thank you! I will be sure to watch out for "input" versus "textarea" going forward!